core/src/main/java/tachyon/worker/hierarchy/StorageDir.java

```diff
@@ -559,29 +642,23 @@ public boolean requestSpace(long size) {
    * @return true if success, false otherwise
    */
   public boolean requestSpace(long userId, long size) {
```

Comment:
it is thread safe.
because mSpaceCounter.requestSpaceBytes and updateUserOwnBytes are thread safe.